pandas/core/indexes/base.py

```diff
@@ -177,10 +181,21 @@ class Index(IndexOpsMixin, PandasObject):
     str = accessor.AccessorProperty(strings.StringMethods)
 
     def __new__(cls, data=None, dtype=None, copy=False, name=None,
-                fastpath=False, tupleize_cols=True, **kwargs):
+                fastpath=False, tupleize_cols=True, names=None,
+                **kwargs):
+
```

Comment:
add a comment that checking name / names

Reply:
Sorry, I didn't get it: a comment for what? Could you, maybe, rephrase your request?

Reply:
you didn't answer my comments. pls add some comments in the code for this section.